qutip/tests/test_qubitcircuit.py

```diff
@@ -224,6 +225,52 @@ def test_add_gate(self):
             # Single control
             pytest.raises(ValueError, qc.add_gate, gate, [0], [1])
 
+        dummy_gate1 = Gate("DUMMY1")
+        inds = [1, 3, 4, 6]
+        qc.add_gate(dummy_gate1, index=inds)
+
+        # Test adding gates at multiple (sorted) indices at once.
+        # NOTE: Every insertion shifts the indices in the original list of
+        #       gates by an additional position to the right.
+        expected_gate_names = [
+            'CNOT',     # 0
+            'DUMMY1',   # 1
+            'SWAP',     # 2
+            'TOFFOLI',  # 3
+            'DUMMY1',   # 4
+            'SWAP',     # 5
+            'DUMMY1',   # 6
+            'SNOT',     # 7
+            'RY',       # 8
+            'DUMMY1',   # 9
+            'RY',       # 10
+        ]
+        actual_gate_names = list(map(attrgetter('name'), qc.gates))
```

Comment:
```suggestion
        actual_gate_names = [gate.name for gate in qc.gates]
```
List comprehension looks a lot clearer.

Reply:
Thanks for reviewing this! Indeed, a list comprehension is clearer in this case. I've updated the PR accordingly.

Reply:
Pleasure! Thank you for making the PR.